src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -1342,6 +1349,9 @@ enum Opcodes
     NUM_MSG_TYPES                                   = 0x51F
 };
 
+typedef Opcodes OpcodeClient;
```

Comment:
This allows to have same code as 6.x while keeping only 1 opcodes enum (at the price of opcode type-safety)